shared/fs/filepreview/index.js

```diff
@@ -0,0 +1,117 @@
+// @flow
+import * as React from 'react'
+import * as Types from '../../constants/types/fs'
+import {globalStyles, globalColors, globalMargins, isMobile} from '../../styles'
+import {Box, Button, Icon, Text} from '../../common-adapters'
+import {formatTimeForMessages} from '../../util/timestamp'
+import FolderHeader from '../header/header-container'
+
+type FilePreviewHeaderProps = {
+  title: string,
+  desc: string,
+}
+
+const FilePreviewHeader = ({title, desc}: FilePreviewHeaderProps) => (
+  <Box>
+    <Box style={filePreviewHeaderStyle}>
+      <Text type="BodyBig">{title}</Text>
+      <Text type="BodySmall">{desc}</Text>
+    </Box>
+  </Box>
+)
+
+type FilePreviewProps = {
+  path: Types.Path,
+  meta: Types.PathItemMetadata,
+}
+
+const humanReadableFileSize = meta => {
```

Comment:
We already have `filesize` in `node_modules`, likely required by some other dependency. So I added it into `package.json` in [my PR](https://github.com/keybase/client/pull/10845/files#diff-83b50b9fdf029c1cf3c705fd882da098R98). It seems better to just use the `filesize` module here.

Reply:
Ah nvm; Nojima requested removing that package: https://github.com/keybase/client/pull/10845#discussion_r174542919

Reply:
Would you mind moving this function into `constants/fs.js`?

Reply:
Done.